cloud-aws-cloudformation/src/main/resources/definitions/aws-environment-minimal-policy.json

```diff
@@ -12,7 +12,10 @@
         "ec2:ReleaseAddress",
         "ec2:DescribeAddresses",
         "ec2:TerminateInstances",
-        "ec2:DeleteSecurityGroup"
+        "ec2:DeleteSecurityGroup",
+        "ec2:ModifyVolume",
+        "ec2:ModifyVolumeAttribute",
+        "ec2:DetachVolume"
```

Comment:
this change should be communicated with PMs since customers are usually using as strict policy as possible

Reply:
communicated to Gaurav